.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+_A brief description of the pull request. Try to keep it non-technical. Please mark your pull request with the appropriate label(s)_
```

Comment:
Do you mean in the PR template? I think this sounds like something that should be prepared elsewhere and linked to maybe. I'd like to keep the template fairly bare-bones.  We can probably remove some of the existing labels.